easybuild/easyconfigs/s/SuiteSparse/SuiteSparse-7.11.0-gfbf-2025b.eb

```diff
@@ -0,0 +1,27 @@
+name = 'SuiteSparse'
```

Comment:
Suitespare has been totally migrated to a CMake project now. I would prefer using generic CMakeMake easyblock (for example: https://github.com/easybuilders/easybuild-easyconfigs/pull/21199)